packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "3.3.2"
```

```suggestion
- version: "3.4.0"
```
Since this is an enhancement, should we consider increasing the minor version instead of the patch version, which is typically for bug fixes?